control_panel_api/tests/test_views.py

```diff
@@ -20,6 +20,26 @@
 )
 
 
+class AssertionsMixin(object):
+    def assert_s3bucket(self, s3bucket):
+        self.assertIn('id', s3bucket)
+        self.assertIn('url', s3bucket)
+        self.assertIn('name', s3bucket)
+        self.assertIn('arn', s3bucket)
+        self.assertIn('created_by', s3bucket)
```

Comment:
This would be more readable as `self.assert('id' in s3bucket)`.

But, is there a way to assert all these fields are present at once? Perhaps using `Set.issubset`?

Reply:
We could use a set.

I disagree on the usage of self.assert.  self.assertIn literally has it in the name of the method, it's intention is clear and is standard practice.  It also goes against previous advice you have given in a PR to use the correct test class assertions (if my memory serves me).

Reply:
Ok, you're right about `assertIn` - it's just a bit weird to read "assert in needle haystack" instead of "assert needle in haystack".

Reply:
👍